testUtils/environmentBuilder.js

```diff
@@ -0,0 +1,21 @@
+export class EnvironmentBuilder {
+  constructor() {
+    this._envs = {};
+    this._envToDelete = [];
+  }
+
+  awsEnvironment = () => {
+    this._envs['LAMBDA_RUNTIME_DIR'] = 'LAMBDA_RUNTIME_DIR';
+    return this;
+  };
+
+  notAwsEnvironment = () => {
+    this._envToDelete.push('LAMBDA_RUNTIME_DIR');
+    return this;
+  };
+
+  applyEnv = () => {
+    process.env = { ...process.env, ...this._envs };
+    this._envToDelete.forEach(env => delete process.env[env]);
+  };
+}
```

Comment:
I think we also need to clean the env after each test.

Reply:
We make this already in `jest.setup.js`